docs/substitutions/global.rst

```diff
@@ -42,8 +42,10 @@
 
 .. |reply_to_msg_id| replace:: If the message is a reply, ID of the original message.
 
-.. |sequenceclassargs| replace:: Accepts any :class:`collections.abc.Sequence` as input instead of just a list. The input is converted to a tuple.
+.. |sequenceclassargs| replace:: |sequenceargs| The input is converted to a tuple.
```

Comment:
did you check if the docs substitute this correctly? cause it's sequenceargs is defined after this, so..

Reply:
Yes, it works